internal/codegen/golang/templated_file.go

```diff
@@ -117,10 +117,16 @@ func (tq TemplatedQuery) EmitParamStruct() string {
 	sb.WriteString("Params struct {\n")
 	typeCol := getLongestInput(tq.Inputs) + 1 // 1 space
 	for _, out := range tq.Inputs {
+		// Name
 		sb.WriteString("\t")
 		sb.WriteString(out.UpperName)
+		// Type
```

Comment:
For the spacing, I calculated in manually for the output struct generation; see the `strings.Repeat`: https://github.com/jschaf/pggen/blob/fe0c28d62d37d84ccd7f2cf48ff34b3a8354b1dc/internal/codegen/golang/templated_file.go#L472

Reply:
@jschaf Please see my approach below, is this okay or would you recommend refactoring it to be some other way?

Reply:
Yep, that's totally fine.

Reply:
Looks like you can directly modify getLongestInput since this is the only call site. That'll fix the lint warning about dead code.